### PR TITLE
refactor(organisms):  SfCollectedProduct

### PR DESCRIPTION
**Pull request by aniamusial**
# Related issue
Awaiting #453 

# Scope of work
- adjusted styles
- added button-text on mobile

Screenshot from 2020-03-09 15-21-08
Screenshot from 2020-03-09 15-20-58

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed

### CODE REVIEW

File: packages/shared/styles/components/organisms/SfCollectedProduct.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@
   position: relative;
   box-sizing: border-box;
   display: flex;
-  max-width: var(--collected-product-max-width, 350px);
+  max-width: var(--collected-product-max-width);
   padding: var(--collected-product-padding, var(--spacer-small));
```
Comment (martaradziszewska) [inline packages/shared/styles/components/organisms/SfCollectedProduct.scss]:
```suggestion
  padding: var(--collected-product-padding, var(--spacer-sm));
```

File: packages/shared/styles/components/organisms/SfCollectedProduct.scss [hunk 2]
```diff
@@ -15,78 +15,75 @@
...
   }
   &__details {
     margin: auto 0 0 0;
     display: flex;
-    justify-content: space-between;
+    justify-content: flex-end;
   }
   @include for-desktop {
+    --collected-product-max-width: 350px;
     --collected-product-padding: 10px;
```
Comment (martaradziszewska) [inline packages/shared/styles/components/organisms/SfCollectedProduct.scss]: can we use here one of the spacers?

File: packages/shared/styles/components/organisms/SfCollectedProduct.scss [hunk 3]
```diff
@@ -15,78 +15,75 @@
     --collected-product-background: var(--c-white);
   }
   &__remove {
-    --button-padding: 0;
-    --icon-size: 0.875rem;
-    --icon-color: var(--c-gray);
+    --icon-size: 8px;
```
Comment (martaradziszewska) [inline packages/shared/styles/components/organisms/SfCollectedProduct.scss]:
```suggestion
    --icon-size: 0.5rem;
```

File: packages/shared/styles/components/organisms/SfCollectedProduct.scss [hunk 4]
```diff
@@ -15,78 +15,75 @@
...
     display: flex;
-    justify-content: space-between;
+    justify-content: flex-end;
   }
   @include for-desktop {
+    --collected-product-max-width: 350px;
     --collected-product-padding: 10px;
     --collected-product-remove-top: 10px;
     --collected-product-remove-right: 0;
     --collected-product-remove-transform: translateX(50%);
```
Comment (martaradziszewska) [inline packages/shared/styles/components/organisms/SfCollectedProduct.scss]: wdyt? we use it very often, maybe we can stick to one form? :)
```suggestion
    --collected-product-remove-transform: translate3d(50%, 0, 0);
```

File: packages/vue/src/components/organisms/SfCollectedProduct/SfCollectedProduct.stories.js [hunk 5]
```diff
@@ -53,15 +53,14 @@ storiesOf("Organisms|CollectedProduct", module)
         :special-price="specialPrice"
     >
       <template #configuration>
-        <div :style="{margin: '20px 0 0 0'}">
+        <div :style="{margin: '14px 0 0 0'}">
```
Comment (pspaczek) [inline packages/vue/src/components/organisms/SfCollectedProduct/SfCollectedProduct.stories.js]: maybe we can set 1rem

File: packages/shared/styles/components/organisms/SfCollectedProduct.scss [hunk 6]
```diff
@@ -15,78 +15,76 @@
...
   }
   &__image {
     background: var(--collected-product-image-background, var(--c-light));
   }
-  & &__quantity-selector {
-    --input-background: var(--c-white);
+  &__quantity-selector {
+    --quantity-selector-background: var(--c-light-variant);
+    --quantity-selector-height: 2.5rem;
+    --quantity-selector-width: 7rem;
```
[the remark this answers is not in the document]
Comment (aniamusial) [reply packages/shared/styles/components/organisms/SfCollectedProduct.scss]: @pspaczek

**Pull request merged**